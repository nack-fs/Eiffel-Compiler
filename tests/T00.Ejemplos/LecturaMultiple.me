//*
Ejemplo LECTURAMULTIPLE.
*//

class LECTURAMULTIPLE;
	
	global
		vars
			i : INTEGER;
			f : DOUBLE;
			c : CHARACTER;

		create
			inicio; 
				
		feature inicio is
			/*local
			do
				println 'I', 'n', 's', 'e', 'r', 't', 'a', 'r', ' ', 'u', 'n', ' ', 'e', 'n', 't', 'e', 'r', 'o', ',', ' ', 'u', 'n', ' ', 'd', 'o', 'u', 'b', 'l', 'e', ' ', 'y', ' ', 'u', 'n', ' ', 'c', 'a', 'r', 'á', 'c', 't', 'e', 'r';
				read i, f, c;
				print 'V','a','l','o','r','e','s',' ','l','e','í','d','o','s',' ','d','e',' ';
				println 'i','=',i,' ','f','=',f,' ','c','=',c;
	end /*inicio
	
end /*LECTURAMULTIPLE

run inicio(); 