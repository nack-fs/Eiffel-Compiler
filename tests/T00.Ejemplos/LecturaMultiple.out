#source "LecturaMultiple.me"

'Declaraciones globales
#global i:int
#global f:float
#global c:char

call main
halt

'**
'** Features
'**

#line 17
inicio:
#func inicio
#ret void

#line 20
pushb 73
outb
pushb 110
outb
pushb 115
outb
pushb 101
outb
pushb 114
outb
pushb 116
outb
pushb 97
outb
pushb 114
outb
pushb 32
outb
pushb 117
outb
pushb 110
outb
pushb 32
outb
pushb 101
outb
pushb 110
outb
pushb 116
outb
pushb 101
outb
pushb 114
outb
pushb 111
outb
pushb 44
outb
pushb 32
outb
pushb 117
outb
pushb 110
outb
pushb 32
outb
pushb 100
outb
pushb 111
outb
pushb 117
outb
pushb 98
outb
pushb 108
outb
pushb 101
outb
pushb 32
outb
pushb 121
outb
pushb 32
outb
pushb 117
outb
pushb 110
outb
pushb 32
outb
pushb 99
outb
pushb 97
outb
pushb 114
outb
pushb 225
outb
pushb 99
outb
pushb 116
outb
pushb 101
outb
pushb 114
outb
pushb 10
outb

#line 21
pusha 0
ini
storei
pusha 2
inf
storef
pusha 6
inb
storeb

#line 22
pushb 86
outb
pushb 97
outb
pushb 108
outb
pushb 111
outb
pushb 114
outb
pushb 101
outb
pushb 115
outb
pushb 32
outb
pushb 108
outb
pushb 101
outb
pushb 237
outb
pushb 100
outb
pushb 111
outb
pushb 115
outb
pushb 32
outb
pushb 100
outb
pushb 101
outb
pushb 32
outb

#line 23
pushb 105
outb
pushb 61
outb
pusha 0
loadi
outi
pushb 32
outb
pushb 102
outb
pushb 61
outb
pusha 2
loadf
outf
pushb 32
outb
pushb 99
outb
pushb 61
outb
pusha 6
loadb
outb
pushb 10
outb

#line 24
ret 0, 0, 0

'**
'** Main
'**

#line 29
main:
#func main
#ret void

#line 29
call inicio

#line 30
ret 0, 0, 0
